components/integrations/add_oauth_app/index.js

```diff
@@ -7,9 +7,8 @@ import {addOAuthApp} from 'mattermost-redux/actions/integrations';
 
 import AddOAuthApp from './add_oauth_app.jsx';
 
-function mapStateToProps(state) {
```

Comment:
We can remove the `mapStateToProps `as we dont need it anymore and pass in null to `connect ` below.

Reply:
fixed it